MODEL_REPO_README_TEMPLATE.md

```diff
@@ -1,17 +0,0 @@
-# MyDrug-Model
```

Comment:
Deleted the README-Template for model repos because it's now part of the model template repository